Hyper/Window.cs

```diff
@@ -9,6 +9,8 @@ namespace Hyper;
 
 internal class Window : GameWindow, IInputSubscriber
 {
+    public static Window Instance;
```

Comment:
since we're using it as a singleton anyway, we could make it a proper singleton

Reply:
But it's not a real singleton since we always call it's constructor in Program.cs and not anywhere else

Reply:
Ok, then it can stay as it is I guess